tests/unittest/llmapi/lora_test_utils.py

```
@@ -11,6 +11,58 @@
+    "Напиши полное предложение, описывающее, что в музее не хватает женских скульптур. Используй фразу \"не хватает\".",
+    "Что означает выражение \"водить за нос\"? Объясни в двух словах.",
+]
+
+
+def _generate_llm_response_lora_fused_modules(llm_class: Type[BaseLLM],
+                                              prompts: list[str],
+                                              **extra_llm_kwargs) -> list[str]:
+    """Generates responses with LoRA requests with the Phi-3-mini-4k-instruct-ru-lora adapter.
+    The used LoRA adapter has fused attention QKV and fused MLP gate up proj modules.
+    Returns the generated texts.
+    """  # noqa: D205
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `D205`)

Remove unused `noqa` directive

(RUF100)